bullet_train/lib/bullet_train/engine.rb

```diff
@@ -21,5 +21,15 @@
 
 module BulletTrain
   class Engine < ::Rails::Engine
+    config.bullet_train = ActiveSupport::OrderedOptions.new
+    config.bullet_train.instrument_callbacks = Rails.env.development? || Rails.env.test?
```

Comment:
I wasn't sure of a better way to ensure we run this instrumentation in our CI tests than to just make it a default in Bullet Train apps' tests too, but that may be a bit too aggressive. @jagthedrummer would you happen to have a better idea?

Reply:
Hey, so sorry for the delay on this! Can we use a config option to toggle this on or off, and then in the starter repo we default to it being on? Directly checking `Rails.env` seems less than ideal.